### PR TITLE
Removed unused strings from resources

### PR DESCRIPTION
**Pull request by YohDeadfall**
Searched in all code files in `src` directory, but haven't found any for removed resources.

### CODE REVIEW

File: src/System.Private.CoreLib/Resources/Strings.resx [hunk 1]
```diff
@@ -1999,36 +1954,12 @@
...
-  </data>
-  <data name="event_TaskStarted" xml:space="preserve">
-    <value>Task {2} executing.</value>
-  </data>
-  <data name="event_TaskWaitBegin" xml:space="preserve">
-    <value>Beginning wait ({3}) on Task {2}.</value>
-  </data>
-  <data name="event_TaskWaitEnd" xml:space="preserve">
-    <value>Ending wait on Task {2}.</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Private.CoreLib/Resources/Strings.resx]: These event_ resources are used EventSource, concatenating "event_" with the event name at run time.

File: src/System.Private.CoreLib/Resources/Strings.resx [hunk 2]
```diff
@@ -2356,30 +2278,6 @@
...
-  <data name="Globalization_cp_28591" xml:space="preserve">
-    <value>Western European (ISO)</value>
-  </data>
-  <data name="Globalization_cp_65000" xml:space="preserve">
-    <value>Unicode (UTF-7)</value>
-  </data>
-  <data name="Globalization_cp_65001" xml:space="preserve">
-    <value>Unicode (UTF-8)</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Private.CoreLib/Resources/Strings.resx]: These resources are all used by Encoding:
https://github.com/dotnet/coreclr/blob/5b1c001bc0fb5bb1ce035c02ec275763f66defc8/src/System.Private.CoreLib/shared/System/Text/EncodingTable.cs#L186

File: src/System.Private.CoreLib/Resources/Strings.resx [hunk 3]
```diff
@@ -1999,36 +1954,12 @@
...
   <data name="CancellationTokenSource_Disposed" xml:space="preserve">
     <value>The CancellationTokenSource has been disposed.</value>
   </data>
   <data name="ConcurrentCollection_SyncRoot_NotSupported" xml:space="preserve">
     <value>The SyncRoot property may not be used for the synchronization of concurrent collections.</value>
   </data>
-  <data name="event_SpinLock_FastPathFailed" xml:space="preserve">
-    <value>SpinLock beginning to spin.</value>
-  </data>
```
Comment (YohDeadfall) [inline src/System.Private.CoreLib/Resources/Strings.resx]: Haven't found this method.

File: src/System.Private.CoreLib/Resources/Strings.resx [hunk 4]
```diff
@@ -1999,36 +1954,12 @@
...
-  <data name="event_SpinWait_NextSpinWillYield" xml:space="preserve">
-    <value>Next spin will yield.</value>
-  </data>
```
Comment (YohDeadfall) [inline src/System.Private.CoreLib/Resources/Strings.resx]: Looks like it isn't used too. Am I right?

File: src/System.Private.CoreLib/Resources/Strings.resx (hunk 3 above)
Comment (stephentoub) [reply src/System.Private.CoreLib/Resources/Strings.resx]: I think you're correct.  Looks like neither SpinLock or SpinWait in corelib are doing any EventSource logging.

File: src/System.Private.CoreLib/Resources/Strings.resx (hunk 4 above)
Comment (stephentoub) [reply src/System.Private.CoreLib/Resources/Strings.resx]: I believe you're correct.

**Comment by YohDeadfall** [comment]
Mostly passed, but there are few infrastructure errors unrelated to the changes.

**Comment by jkotas** [comment]
Thank you!

**Pull request merged**